src/layouts/CoreLayout/styles.scss

```diff
@@ -1,3 +1,10 @@
 .mainContainer {
```

Comment:
We'll probably want to make this container a separate styled-component; I can refactor this before merging.

Reply:
If you see changes that should belong in the base `recipe-login` I think everyone would be fine with getting the changes in there then merge into here

Reply:
Or y'know depending on how far back the changes go, back to strangeluv vanilla

Reply:
That makes sense! Are we planning on merging `recipe-login-material` into anything, or is this PR just for illustrative purposes? I do like the idea of keeping a totally style-less `recipe-login` for non-Material solutions.

Reply:
Yeah I _think_ that's the plan -- we will most likely be starting with `recipe-login-material` on a new project but yeah keeping the material out of `recipe-login` I think is the goal

Reply:
Oh good call, I forgot this class and stylesheet exists lol

This branch and `recipe-login` will live separately (I believe), and this PR is just for review and discussion.